manim/scene/stream_file_writer.py

```diff
@@ -0,0 +1,106 @@
+from abc import ABCMeta
+
+import os
+import subprocess
+
+from .. import config, logger
+from ..constants import FFMPEG_BIN
+from .scene import Scene
+from .scene_file_writer import SceneFileWriter
+from ..utils.file_ops import guarantee_existence
+
+
+class StreamFileWriter(SceneFileWriter):
+    def __init__(self, renderer):
+        super().__init__(renderer, "")
+        vars(self).update(config.streaming_config)
+        path = os.path.join(config.get_dir("streaming_dir"), "clips")
+        self.FOLDER_PATH = os.path.relpath(guarantee_existence(path))
+        # To prevent extensive overwriting
+        self.partial_movie_directory = self.FOLDER_PATH
+
+    def init_output_directories(self, scene_name):
+        """The original :class:`SceneFileWriter` uses this method while initializing.
+        I need most of that initialization, minus this. Hence kicked to the curb.
+        """
+        pass
+
+    @property
+    def file_path(self):
+        return self.partial_movie_files[-1]
+
+    def end_animation(self, allow_write=False):
+        """The point in the animation where the file exists."""
+        super().end_animation(allow_write=allow_write)
+        self.stream()
+
+    def combine_movie_files(self):
+        """Also to reduce overriding code."""
+        pass
+
+    def stream(self):
+        logger.info(
+            "Houston, we are ready to launch. Sending over to %(url)s",
+            {"url": {self.streaming_url}},
+        )
+        command = [
+            FFMPEG_BIN,
+            "-re",
+            "-i",
+            self.file_path,
+            "-vcodec",
+            "copy",
+            "-an",
+            "-loglevel",
+            "quiet",
+        ]
+
+        if self.streaming_protocol == "rtp":
+            command += ["-sdp_file", self.sdp_path]
+        command += [
+            "-f",
+            (
+                self.streaming_protocol
+                if self.streaming_protocol == "rtp"
+                else "mpegts"
+            ),  # udp protocol didn't work for me but if it does for you congrats
```

Comment:
remove before merge

Reply:
Fine fine. RTP be default.
I will have to mention in the PR about glitching and all that